pyproject.toml

```diff
@@ -17,7 +17,7 @@ python = "^3.10"
 
 [tool.poetry.build]
 generate-setup-file = false
-script = "build.py"
+script = "custom_build.py"
```

Comment:
Rename `build.py` to `custom_build.py` to avoid conflict
<img width="700" alt="image" src="https://github.com/user-attachments/assets/34f08e4b-804f-4372-8306-e3e414a1a3c1" />

This resolved my installation problems in conda, and avoided the use of `pip install -e .`

Reply:
What was your installation setup for conda like? Can the built wheel not be installed directly?

Reply:
Here is my screen record(Python3.10, Debian:11):
![p2g](https://github.com/user-attachments/assets/c2e164f4-e429-4102-b34e-68f7d86340d6)

Might be related to this issue: https://github.com/python-poetry/poetry/issues/7576#issuecomment-1458251330


**Note**: the `build.py` works well in my other server(ubuntu22). Maybe it is not caused by conda.